recipe/meta.yaml

```diff
@@ -11,19 +11,19 @@ source:
   sha256: 7e1e5b56cc735432a7369cbfa0efe50fa113ebecdc04ae6922deba8b84582d0c
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
     - setuptools
   run:
-    - python >=3.7
+    - python >=3.8
   run_constrained:
-    - cryptography >=3.3.1
+    - cryptography >=3.4.0
```

Comment:
This constraint being incorrect can cause problems downstream (e.g. [globus-sdk](https://github.com/conda-forge/globus-sdk-feedstock)), which relies on `pyjwt[crypto]`.

Reply:
I see that this is likely a moot point because cryptography 3.4.0 was from over 3 years ago so the odds of either of these very old versions getting installed are next to nil.